sql/moz-fx-data-shared-prod/hubs/active_subscription_ids/view.sql

```diff
@@ -0,0 +1,23 @@
+CREATE OR REPLACE VIEW
+  `moz-fx-data-shared-prod.hubs.active_subscription_ids`
+AS
+WITH max_active_date AS (
+  SELECT AS VALUE
+    MAX(active_date)
+  FROM
+    `moz-fx-data-shared-prod`.hubs_derived.active_subscription_ids_v1
+)
+SELECT
+  active_subscription_ids_live.*
+FROM
+  `moz-fx-data-shared-prod`.hubs_derived.active_subscription_ids_live
```

Comment:
```suggestion
  `moz-fx-data-shared-prod`.hubs_derived.active_subscription_ids_live AS active_subscription_ids_live
```

The stage deploy replaces the project with something like `bigquery-etl-integration-test.hubs_derived_test.active_subscription_ids_live` which encapsulates the entire identifier in "`".

Reply:
Thanks to your quick fix in #3636 this workaround won't be necessary. 🙇